doc/quickstart.md

```diff
@@ -0,0 +1,111 @@
+# vfkit quick start
+
+## Introduction
+
+vfkit is a macOS command line hypervisor.
+It uses [Apple's Virtualization Framework](https://developer.apple.com/documentation/virtualization?language=objc) to run virtual machines.
```

Comment:
and continue the sentence with
`, which uses ...`

Reply:
Done.